### PR TITLE
Fix deployment of Oppia-ml on GCE

### PR DESCRIPTION
**Pull request by prasanna08**
Fix files so that oppia-ml can be deployed cleanly.

### CODE REVIEW

File: core/services/remote_access_services.py [hunk 1]
```diff
@@ -47,17 +47,17 @@ def _get_vm_id():
         return vmconf.DEFAULT_VM_ID
 
     # Get VMID dynamically from metadata.
-    return metadata_services.get_metadata_param(
-        vmconf.METADATA_VM_ID_PARAM_NAME)
+    return str(metadata_services.get_metadata_param(
```
Comment (seanlip) [inline core/services/remote_access_services.py]: why need to cast?

**Review by seanlip — APPROVED** [review]
All looks good. Just one question. Thanks!

File: core/services/remote_access_services.py (hunk 1 above)
Comment (prasanna08) [reply core/services/remote_access_services.py]: [`HMAC`](https://stackoverflow.com/questions/20849805/python-hmac-typeerror-character-mapping-must-return-integer-none-or-unicode) does not support unicode strings.

Comment (seanlip) [reply core/services/remote_access_services.py]: Add a comment then, here and below.

Comment (prasanna08) [reply core/services/remote_access_services.py]: Done.

**Pull request merged**